src/fix/main.c

```diff
@@ -1401,7 +1401,6 @@ do { \
 			break;
 
 		default:
-			fprintf(stderr, "FATAL: unknown option '%c'\n", ch);
```

Comment:
I don't think it's good UX to print out the usage without saying why we're doing so

Reply:
Okay, I've added this to all the programs.